internal/health/status/check.go

```diff
@@ -90,10 +91,12 @@ func (c *Checker) executeConditionChecks(ctx context.Context, etcd *druidv1alpha
 		wg.Wait()
 	})()
 
-	results := make([]condition.Result, 0, len(ConditionChecks))
+	results := make([]condition.Result, 0, len(ConditionChecks)+1)
 	for r := range resultCh {
 		results = append(results, r)
 	}
+	backupReadyChecker := condition.BackupReadyCheck(c.cl, results)
```

Comment:
i really do not understand this. You removed it from `ConditionChecks` but introduce this here? why?